integration-tests/contract-permits.spec.ts

```diff
@@ -78,13 +78,13 @@ CONFIGS().forEach(({ lib, rpc, setup, createAddress }) => {
   const Tezos = lib;
   Tezos.setPackerProvider(new MichelCodecPacker());
 
-  describe(`Test of contracts having a permit for tzip-17: ${rpc}`, () => {
+  describe(`Test contract origination having a permit for tzip-17 through contract api: ${rpc}`, () => {
```

Comment:
It is not only the origination that is tested

Reply:
changed to "Verify contract origination, transfer, and minting with a permit for tzip-17 through contract API:"